pkg/api/v1alpha1/cluster_types.go

```diff
@@ -410,6 +410,25 @@ func (n *KindnetdConfig) Equal(o *KindnetdConfig) bool {
 	return true
 }
 
+func UsersSliceEqual(a, b []UserConfiguration) bool {
```

Comment:
Not sure if this makes sense under cluster. Would think under `machineconfig_types.go`, but am okay with this.